docs/get-details/transactions/index.md

```diff
@@ -675,19 +675,56 @@ Here are three example scenarios and how the round range may be calculated for e
+There are two primary ways to set fees on a transaction.
 
-In the future as more transactions are added to the network (and blocks are full), the minimum transaction fee may not guarantee that your transaction is processed as quickly as other transactions with higher fees set. In this case, using the returned suggested fee, which is based on the current transaction load, is the preferred method.
+## Suggested Fee
+
+The SDK provides a method to get the suggested parameters from an the algod REST server, including the [**suggested fee** per byte (`fee`)](../../rest-apis/algod/v2#transactionparams) which can be used to set the total fee for a transaction. This value is multiplied by the estimated size of the transaction in bytes to determine the total transaction fee. If the result is less than the minimum fee, the minimum fee is used instead. 
+
+When using the SDK to build a transaction, if the suggested parameters are passed to one of the helper methods, the fee for the transaction will be set based on the above computation.
 
 ## Flat Fee
-You can also manually set a **flat fee**. If you choose this method, make sure that your fee covers at least the [minimum transaction fee (`minFee`)](../../reference/rest-apis/algod/v1#transactionparams), which can be obtained from the suggested parameters method call in each of the SDKs.  Flat fees may be useful for applications that want to guarantee showing a specific rounded fee to users or for a transaction that is meant to be sent in the future where the network traffic conditions are unknown.
+You can also manually set a **flat fee**. If you choose this method, make sure that your fee covers at least the [minimum transaction fee (`min-fee`)](../../rest-apis/algod/v2#transactionparams), which can be obtained from the suggested parameters method call in each of the SDKs.  Flat fees may be useful for applications that want to guarantee showing a specific rounded fee to users or for a transaction that is meant to be sent in the future where the network traffic conditions are unknown.
+
+# Pooled transaction fees
+
+The Algorand protocol supports pooled fees, where one transaction can pay the fees of other transactions within the same atomic group. 
+
+For atomic transactions, the fees set on all transactions in the group are summed. This sum is compared against the protocol determined expected fee for the group and may proceed as long as the sum of the fees is at least the required fee for the group.
+
+<center>![Atomic Pooled Fees](../imgs/atomic_transfers-3.png)</center>
+<center>*Atomic Pooled Fees*</center>
+
+!!! note
+    [Inner transactions](/get-details/dapps/smart-contracts/apps/#inner-transactions) may have their fees covered by the outer transactions but they may not cover outer transaction fees.
```

Comment:
@jannotti Since now multiple level of inner transactions may appear, is the restriction that an inner transaction cannot pay for the fee of the outer transaction or just the top-level transaction?

Concretely, suppose tx1 makes a transaction tx2 that makes a transaction tx3.
Can we have:
* tx1 fee is 0.001
* tx2 fee is 0
* tx3 fee is 0.002

Reply:
No. At the time that tx2 is constructed (maybe as part of a group), the existing fee credit plus the fees of tx2 (and _siblings_, not _children_) must cover the cost of tx2's group. Processing will not continue in the hopes that tx2 will construct a tx3 with a big fee.